server.py

```diff
@@ -31,18 +35,20 @@ def server():
         server_socket.close()
 
 
-def response_ok(msg):
-    result = "HTTP/1.1 200 OK\r\n"
-    con_type = "Content-Type: text/plain\r\n"
-    body = "Content length: {}".format(len(msg))
-    return "{}{}{}".format(result, con_type, body)
+def response_ok(msg, resolved):
+    result = "\nHTTP/1.1 200 OK\r\n"
+    timestamp = 'Date: {}\r\n'.format(email.utils.formatdate(usegmt=True))
+    con_type = "Content-Type: {}".format(msg)
+    body = "{}".format(resolved)
+    return "{}{}{}{}".format(result, timestamp, con_type, body)
 
 
 def response_error(error_code, error_msg):
     error_type = "HTTP/1.1 {} ERROR\r\n".format(error_code)
+    timestamp = 'Date: {}\r\n'.format(email.utils.formatdate(usegmt=True))
     con_type = "Content-Type: text/plain\r\n"
     body = "ERROR {}, {}\r\n".format(error_code, error_msg)
-    return "{}{}{}".format(error_type, con_type, body)
+    return "{}{}{}{}".format(error_type, timestamp, con_type, body)
```

Comment:
same message as for response_ok.  Without a blank line between headers and body this is *not* an HTTP response.